src/LivewireManager.php

```diff
@@ -177,7 +177,15 @@ protected function javaScriptAssets($jsFileName, $options)
     {
         $jsonEncodedOptions = $options ? json_encode($options) : '';
 
-        $appUrl = rtrim($options['base_url'] ?? '', '/');
+        $appUrl = '';
+
+        if (file_exists(config_path('livewire.php'))) {
```

Comment:
rather than checking for the existence of the config file, maybe just use the aforementioned second param of the config helper for a default. It's more flexible that way (thinking things like testing).

Reply:
When I thought this through before it made sense to me to do it this way for some reason, will review.